common/migration/1682516060693-suffix-participants-and-organizers-with-ids.ts

```diff
@@ -0,0 +1,20 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class suffixParticipantsAndOrganizersWithIds1682516060693 implements MigrationInterface {
+    name = 'suffixParticipantsAndOrganizersWithIds1682516060693';
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "lecture" DROP COLUMN "organizers"`);
+        await queryRunner.query(`ALTER TABLE "lecture" DROP COLUMN "participants"`);
```

Comment:
Are these new columns or do they already have data in prod?

Reply:
They shouldn't. I added them with a PR yesterday and they shouldn't be currently in use.